.github/workflows/continuous-integration-workflow.yml

```diff
@@ -58,5 +58,5 @@ jobs:
       - name: Test install
         working-directory: example/tutorial/01_hello_world
         run: |
-          cmake -B builddir
+          cmake -B builddir -DCMAKE_CXX_COMPILER=${{ matrix.cxx }}
```

Comment:
@dalg24 that hints towards an issue, i think we should warn users if they try to using a different compiler than what kokkos was compiled with.

Reply:
That probably should be discussed. Do you want to open an issue for that?

Reply:
Meh, I leave that to you ;-)